lib/elixir/test/elixir/kernel/warning_test.exs

```diff
@@ -829,6 +829,33 @@ defmodule Kernel.WarningTest do
     end) =~ "key :__struct__ is ignored when using structs"
   end
 
+  test "catch comes before rescue in try block" do
+    output_with_warning = capture_err(fn ->
+      Code.eval_string """
+      try do
+        :trying
+      catch
+        _ -> :caught
+      rescue
+        _ -> :error
+      end
+      """
+    end)
+    assert output_with_warning =~ ~s("catch" should always come after "rescue" in a try/do block)
+    output_withot_warning = capture_err(fn ->
```

Comment:
Typo on "without". :)

Reply:
Although we typically don't test things do not emit a warning. Especially in this case, since we would get warnings all over the language, so I would say we can safely remove this assertion.

Reply:
I think we usually don't test the lack of a warning. 🤔

Reply:
oops :smile: